pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -22,7 +24,17 @@ class Range(Module):
   is_original_torch_module = False
 
   def create_returnn_layer_dict(self, limit, start, delta, dtype, sparse=False) -> Dict[str, Any]:
-    return {"class": "range", "limit": limit, "start": start, "delta": delta, "dtype": dtype, "sparse": sparse}
+    if isinstance(limit, SizeValue):
+      limit = limit.as_tensor()
+      return {"class": "range_from_length", "from": self._get_input_layer_name(limit)}
+    else:
+      return {"class": "range", "limit": limit, "start": start, "delta": delta, "dtype": dtype, "sparse": sparse}
+
+  def _get_output_shape_from_returnn(self, inputs_flat: List, layer: LayerBase
```

Comment:
I assume `List[Tensor]`?

Reply:
Actually no. Right now, we get whatever the input args were in `arange`, so a `SizeValue` and `int`s. This is a problem, right? The crash in the test case where the `feed_dict` is not prepared correctly is related to this, I think.